server/controllers/list.js

```diff
@@ -144,12 +159,25 @@ const addItemToList = (req, resp) => {
       }
 
       doc
-        ? resp.status(200).send(item)
+        ? resp.status(200).send(itemToSend(item, userId))
         : resp.status(404).send({ message: 'List  not found.' });
     }
   );
 };
 
+const getItemsForList = (userId, list) => {
```

Comment:
I wonder if that method should be place in here. It is not a controller in fact. It probably could be defined as instance or static method for its model. Or maybe it could be placed in utilities folder like in user's helper methods case.
https://mongoosejs.com/docs/2.7.x/docs/methods-statics.html
https://mongoosejs.com/docs/api.html#schema_Schema-method

Reply:
What for is the `for` in name? Could it be simpler like `getListItems` or even `listItems`?